twolevel_dbm.sh

```
@@ -233,6 +233,7 @@ i=1
 while read -r subject_scans; do
 
   IFS=',' read -r -a scans <<<${subject_scans}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)